azure/azure_kv.go

```diff
@@ -109,7 +132,7 @@ func (az *azureSecrets) GetSecret(
 		return nil, secrets.NoVersion, err
 	}
 
-	secretResp, ok := resp.(keyvault.SecretBundle)
+	secretResp, ok := resp.(azsecrets.GetSecretResponse)
```

Comment:
Azure now returns secret version as well.
can we return version as well?

```go
secretResp.ID.Version()
```

Reply:
AWS Secrets Manager returns the version.The only reason all providers have not been updated to return the version is lack of bandwidth to check and test if each one of them If we are modifying this provider and you are able to verify its correctness we can modify it